app/Domain/Integrations/Controllers/IntegrationController.php

```diff
@@ -22,9 +22,9 @@
 final class IntegrationController extends Controller
 {
     public function __construct(
-        private readonly SubscriptionRepository $subscriptionRepository,
-        private readonly IntegrationRepository $integrationRepository,
-        private readonly CurrentUser $currentUser
+        private readonly SubscriptionRepository        $subscriptionRepository,
+        private readonly EloquentIntegrationRepository $integrationRepository,
```

Comment:
Should this inject the interface?

I also see some auto-indentation from PHPStorm

Reply:
I decided not to refactor classes that were out of scope to use the interface instead of the concrete class, I think that's something better suited for a separate PR (also to add interfaces to the other repositories)